extractor2/README.md

```diff
@@ -0,0 +1,23 @@
+# Berserker Extractor2
```

Comment:
Why don't just replace the previous extractor? I don't think we want to maintain two versions of it, even before the first stable release.